docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
     working_dir: /src
     volumes:
       - $PWD:/src
+      - $PWD/dist/terraform-provider-conjur_linux_amd64/:/usr/share/terraform/plugins/terraform.example.com/cyberark/conjur/0.4.0/linux_amd64/
```

Comment:
same Q here - hardcoding the version seems like it will be hard to maintain going forward